apps/modernization-ui/src/design-system/entry/multi-value/RepeatingBlock.tsx

```diff
@@ -31,7 +31,7 @@ export const RepeatingBlock = <V extends FieldValues>({
     formRenderer,
     viewRenderer
 }: Props<V>) => {
-    const form = useForm<V>({ mode: 'onSubmit', defaultValues });
+    const form = useForm<V>({ defaultValues });
```

Comment:
Question: What happens when you click `Add` in an invalid state?

Reply:
I think your referring to how by default the form is already using onSubmit so we don't have to specify it? so when we click save the button would then handle that because prior to that it would be disabled right?